sound/soc/sof/control.c

```diff
@@ -79,7 +82,7 @@ static void snd_sof_refresh_control(struct snd_sof_control *scontrol)
 
 	/* set the ABI header values */
 	cdata->data->magic = SOF_ABI_MAGIC;
```

Comment:
for ease in the future for changing ABI, we should probably use an "init_ipc" function so we only have to change in one place